addons/tools/jq/package.mk

```diff
@@ -0,0 +1,51 @@
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
+#
+#  OpenELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  OpenELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="jq"
+PKG_VERSION="1.4"
+PKG_REV="1"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="http://stedolan.github.io/jq/"
+PKG_URL="https://github.com/stedolan/$PKG_NAME/archive/$PKG_NAME-$PKG_VERSION.zip"
```

Comment:
PKG_URL="http://stedolan.github.io/jq/download/source/$PKG_NAME-$PKG_VERSION.tar.gz" and delete unpack() below.